pkg/vulnsrc/vulnerability/vulnerability_test.go

```diff
@@ -56,12 +61,21 @@ func TestGetDetail(t *testing.T) {
 			},
 			expectedSeverity:       types.SeverityMedium,
 			expectedVendorSeverity: types.VendorSeverity{"redhat": 4, "ubuntu": 1, "rust-advisory-db": 4},
-			expectedTitle:          "test vulnerability",
-			expectedDescription:    "a test vulnerability where vendor rates it lower than NVD",
-			expectedRefs:           []string{"http://foo-bar.com/baz"},
+			expectedVendorVectors: types.VendorVectors{
+				RedHat: types.CVSSVector{
+					V2: "AV:N/AC:M/Au:N/C:N/I:P/A:N",
+					V3: "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
+				},
+				Ubuntu: types.CVSSVector{
+					V3: "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
+				},
```

Comment:
nit: `ubuntu` doesn't have CVSS vector, right? It's ok since this is a test, but someone perhaps would misunderstand that ubuntu should return CVSS vector, looking at this expectation. I prefer using actual data, but this is nit-pick.

Reply:
Yeah I see your point. Although it's correct what you say since Ubuntu doesn't have vectors, I only added this because I would like to have more than once vendor to play with in a test.